tests/post/steps/test_volume.py

```diff
@@ -107,3 +118,19 @@ def _check_size():
     )
 
 # }}}
+# Helpers {{{
+
+def _create_volume(k8s_client, body):
+    try:
+        k8s_client.create_cluster_custom_object(
+            group="storage.metalk8s.scality.com",
+            version="v1alpha1",
+            plural="volumes",
+            body=yaml.safe_load(body)
+        )
+    except (ApiException, HTTPError) as exc:
+        if isinstance(exc, ApiException) and exc.status == 404:
+            assert False, 'Exception while creating volume {}'.format(name)
```

Comment:
While only error when 404? Other errors are also errors.

Should be defined as:

```python
def _create_volume(k8s_client, body):
    k8s_client.create_cluster_custom_object(
        group="storage.metalk8s.scality.com",
        version="v1alpha1",
        plural="volumes",
        body=yaml.safe_load(body)
    )
```